Makefile

```diff
@@ -256,6 +257,26 @@ run-in-kind-no-controller: IMAGE_TAG=test
 run-in-kind-no-controller: SKIP_CONTROLLER_BUILD=true
 run-in-kind-no-controller: load-images-to-kind deployment-config-no-controller deploy-current-config ## Build and deploy porch without the controllers into a kind cluster
 
+.PHONY: run-default
```

Comment:
but the simplest solution IMO is to undeploy the porch instance that was deployed by the ansible playbook (call `kpt live destroy /tmp/kpt-pkg/nephio/core/porch`) and then reinstall it with `make run-in-kind`. and then use the same development process that is currently documented.

Reply:
> but the simplest solution IMO is to undeploy the porch instance that was deployed by the ansible playbook (call `kpt live destroy /tmp/kpt-pkg/nephio/core/porch`) and then reinstall it with `make run-in-kind`. and then use the same development process that is currently documented.

The problem there is that if you uninstall, it undefines all the package variants and packagerevisions that are put in place by the Nephio install.

Reply:
> Couldn't we achieve the same effect by something similar to the following process:
> 
>     * add these lines to the bashrc of the sandbox VM:
> 
> 
> ```
> export DEPLOYPORCHCONFIGDIR=/tmp/kpt-pkg/nephio/core/porch
> export KIND_CONTEXT_NAME=<whatever is the name of the kind cluster in the Nephio sandbox :)>
> ```
> 
>     * reuse the existing make targets: `run-in-kind-*`

The problem I had was that I couldn't get `kpt` to play nicely with the old and new package. I tried to use the existing targets but because the underlying kpt wouldn't work, they failed. Maybe you could take a look @kispaljr

Reply:
> > but the simplest solution IMO is to undeploy the porch instance that was deployed by the ansible playbook (call `kpt live destroy /tmp/kpt-pkg/nephio/core/porch`) and then reinstall it with `make run-in-kind`. and then use the same development process that is currently documented.
> 
> The problem there is that if you uninstall, it undefines all the package variants and packagerevisions that are put in place by the Nephio install.

OK, I will try this out

Reply:
By the way a tiny opinion, is it possible to use `_` like `DEFAULT_PORCH_CONFIG_DIR` it increases the readability.

Reply:
This is all changed int he new version of the PR.